docs/PacketFence_Aerohive_Quick_Install_Guide.asciidoc

```diff
@@ -18,433 +18,267 @@ include::includes/global-attributes.asciidoc[]
 
-Almost all of the required information here are self-explanatory. The only one that could be confusing is the DHCP Servers section. In this one, enter a comma-delimited list of all the DHCP Server on the customer network so when PacketFence will see DHCP traffic originating from these IPs, no rogue-dhcp alerts will be triggered.
-
-Click 'Continue' once all the fields are completed.
-
-Step 5: Administration
-++++++++++++++++++++++
-This is the step where you create the administrative user to access the PacketFence Administration Web Interface.
-
-Simply provide the desired username and password, then click 'Create user'.
-
-Step 6: Services - Confirmation
-+++++++++++++++++++++++++++++++
-The last but not the least. Here, you start the PacketFence server according to the configurations made in the previous steps. If everything goes as expected, you'll be prompted by a window inviting you to continue to the web administration interface.
-
-You'll be able to login to the PacketFence web administration interface with the credentials created in Step 4.
-
-Services status will help you monitor if everything goes as expected. If not, you'll see which service is in trouble and the log output will help you determine the problem that occurs.
+Step 2: Configure Network and PacketFence
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Configuring Aerohive access point in PacketFence
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Now that you have a fully functional PacketFence installation, you'll need to add the Aerohive AP to the PacketFence switches database for correct integration.
-
-To do so, login to the PacketFence web administration interface if it is not already done. Click on the 'Configuration' tab and select the Switches section.
-
-We'll use the clone function to add a new switch (the AeroHive AP) to the PacketFence database. With the mouse pointer, go over the default switch and you'll see a set of icons appearing at the left of it. Click on the second one (a paper sheet with a plus sign).
-
-Configure the switch like the following:
-
-image::docs/images/aerohive-pf-config-webauth.png[scaledwidth="100%",alt="General switch settings"]
-
-The section Roles should be empty while doing webauth.
-
-image::docs/images/aerohive-pf-config-snmp.png[scaledwidth="100%",alt="SNMP switch settings"]
-
-Under the tab RADIUS, please set your RADIUS secret.
-
-Click 'Add switch'
-
-The newly added access point should show up in the list.
-
-PacketFence configuration is done. You may now reboot the demonstration PC (leave the USB key in). Once rebooted, the web browser should open in the PacketFence web administration interface.
 
-Step 3: Configure Aerohive
-~~~~~~~~~~~~~~~~~~~~~~~~~~
-This section will help you configure your Aerohive AP for integration with PacketFence using the VLAN-based enforcement.
+We will nowneed to create a new switch, in PacketFence to be able to manage the terminals behind the Aerohive APs.
 
```

Comment:
now need, remove the comma
to manage the terminals behind the Aerohive APs. -> to manage endpoints connecting to the Aerohive SSIDs.